### PR TITLE
post sign motion implemented

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

**Comment by evidentti** [comment]
Further implementation is pending backend updates.

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 1]
```diff
@@ -363,7 +366,7 @@ angular.module('dashboard')
                     $log.log("meetingStatusCtrl.getMotions done", resp);
                     resultData.objects = angular.isArray(resp) ? resp : [];
                 }, function (error) {
-                    $log.error("meetingStatusCtrl.getMotions ", arguments);
+                    $log.error("meetingStatusCtrl.getMotions ", error);
```
Comment (tkenakka) [inline app/scripts/controllers/meeting.status.ctrl.js:369]: Interesting, what was the problem with `arguments` ?

File: app/scripts/motions.directive.js [hunk 2]
```diff
@@ -13,18 +13,58 @@
...
+                        else {
+                            aMotion.supporters = [supporter];
+                        }
+                    }
+                    else {
+                        $log.error("dbMotions.sign done ", resp);
+                    }
+                }, function (error) {
+                    $log.error("dbMotions.sign ", error);
+                    self.error = { 'errorCode': error.errorCode, 'errorString': Utils.stringIdForError(error.errorCode) };
```
Comment (tkenakka) [inline app/scripts/motions.directive.js:60]: This is OK by me, but just remarking how there's `GENERALERROR` in `CONST` which could be extended by `errorStringId` and/or `errorString` and reused.

**Review by tkenakka — APPROVED** [review]
LGTM

**Pull request merged**